lib/akamai_rspec/request.rb

```diff
@@ -82,9 +82,12 @@ def get(url, headers = {})
       req['Host'] = uri.hostname
       uri.hostname = @domain
 
-      response = Net::HTTP.start(uri.hostname, uri.port) do |http|
-        http.request(req, nil) { |http_response| http_response }
+      net = Net::HTTP.new(uri.hostname, uri.port)
+      if uri.scheme == 'https'
+        net.use_ssl = true
+        net.verify_mode = OpenSSL::SSL::VERIFY_NONE
```

Comment:
SSL verification will never work because the target hostname (i.e www.example.com.edgkey.net) is never going to be listed as a Subject Alternate Name in the certificate served.